src/models/OAuthModel.php

```diff
@@ -167,11 +167,9 @@ public function newAccessToken($consumer_key, $user_id)
     {
         $hash              = $this->generateToken();
         $accessToken       = substr($hash, 0, 16);
-        $accessTokenSecret = substr($hash, 16, 16);
 
         $sql = "INSERT INTO oauth_access_tokens set
                 access_token = :access_token,
-                access_token_secret = :access_token_secret,
```

Comment:
Is that necessary for the talk links? Otherwise please add that as a further PR. 

And even though it's not used ATM it might become necessary in the future.